src/events.ts

```diff
@@ -123,7 +124,7 @@ const messageBuilder = (context, event: EventResult): string => {
 };
 
 const isStrict = (event: Event): boolean => {
-  if (event.strict == undefined && event.threshold != undefined) {
+  if (event.strict == undefined || event.threshold != undefined) {
```

Comment:
I am unsure about this. Did you check the rest of tests (assets) still properly work with this change?

Reply:
otherwise `threshold` wont work

Reply:
if strict = true, and threshold something, threshold wont work